app/Http/Controllers/NoteController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\FormNoteRequest;
+use Illuminate\Http\Request;
+use App\Models\Note;
+use App\Models\User;
+use Illuminate\Contracts\Session\Session;
+
+class NoteController extends Controller
+{
+    public function index(Request $request){
+        if($request->session()->get('user')!=null){
+            $notes=Note::where('user_id',$request->session()->get('user')->id)->get();
+        }else{
+            $email=$request->cookie('email');
+            $password=$request->cookie('password');
+            $user=User::where('email',$email)->where('password',$password)->first();
+            $notes=Note::where('user_id',$user->id)->get();
+        }
```

Comment:
```suggestion
            $notes = Note::where('user_id', $request->session()->get('user')->id)->get();
        } else {
            $email = $request->cookie('email');
            $password = $request->cookie('password');
            $user = User::where('email', $email)->where('password', $password)->first();
            $notes = Note::where('user_id', $user->id)->get();
        }
```
Mấy chỗ này cách ra nhé :3 fix cả các chỗ khác nữa nha